src/System.Threading/tests/System.Threading.Tests.builds

```diff
@@ -9,7 +9,7 @@
     </Project>
     <Project Include="System.Threading.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TestTFMs>net46</TestTFMs>
+      <TestTFMs>net463</TestTFMs>
```

Comment:
I think that apart from this change, we should keep a configuration for net46 since actually that is the one that runs on Helix today, so basically just add the following lines to this .builds file:
```xml
<Project Include="System.Threading.Tests.csproj">
      <TargetGroup>netstandard1.3</TargetGroup>
      <TestTFMs>net46</TestTFMs>
</Project>
```